plugins/com.google.cloud.tools.eclipse.appengine.login/plugin.xml

```diff
@@ -1,5 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+   <extension
+         point="org.eclipse.ui.commands">
+      <category
+            id="com.google.cloud.tools.eclipse.appengine.login.commands.category"
+            name="Google Login Category">
+      </category>
+      <command
+            categoryId="com.google.cloud.tools.eclipse.appengine.login.commands.category"
+            id="com.google.cloud.tools.eclipse.appengine.login.commands.loginCommand"
+            name="Google Login Command">
+      </command>
+   </extension>
+   <extension
+         point="org.eclipse.ui.handlers">
+      <handler
+            class="com.google.cloud.tools.eclipse.appengine.login.GoogleLoginCommandHandler"
+            commandId="com.google.cloud.tools.eclipse.appengine.login.commands.loginCommand">
+      </handler>
+   </extension>
+   <extension
+         point="org.eclipse.ui.menus">
+      <menuContribution
+            locationURI="menu:file?after=additions">
+         <command
+               commandId="com.google.cloud.tools.eclipse.appengine.login.commands.loginCommand"
+               id="com.google.cloud.tools.eclipse.appengine.login.menus.loginCommand"
+               label="%loginMenuItemLabel"
+               mnemonic="L">
```

Comment:
It conflicts with 'Close All' (although a conflict itself won't be a problem).

Let me then go with G? It doesn't conflict.